controllers/reconcile_test.go

```diff
@@ -858,6 +858,9 @@ func TestOpenTelemetryCollectorReconciler_VersionedConfigMaps(t *testing.T) {
 			Namespace: metav1.NamespaceDefault,
 		},
 		Spec: v1beta1.OpenTelemetryCollectorSpec{
+			OpenTelemetryCommonFields: v1beta1.OpenTelemetryCommonFields{
+				PodDisruptionBudget: &v1beta1.PodDisruptionBudgetSpec{},
+			},
```

Comment:
This is just to avoid logs about the default being set. We should really get rid of that log line imo.